en/documentation/index.md

```diff
@@ -241,6 +241,6 @@ These links were more prominent but haven't been updated in ages.
 [ruby-lsp]: https://github.com/Shopify/ruby-lsp
 [rubyapi-org]: https://rubyapi.org/
 [rubyin20]: https://www.ruby-lang.org/en/documentation/quickstart/
-[shopify_lsp]: https://shopify.github.io/ruby-lsp/editors.html)
+[shopify_lsp]: https://shopify.github.io/ruby-lsp/editors.html
```

Comment:
I know this will be different fix, but IMO this should be called `ruby-lsp-supported-editors` and be placed under `ruby-lsp` (line 242).

Reply:
fixed 2c82d0b83f6bd00e668f5e3567388